EventFilter/L1TRawToDigi/plugins/implementations_stage2/EtSumUnpacker.cc

```diff
@@ -91,6 +91,15 @@ namespace stage2 {
 
        res_->push_back(bx,ht);
 
+
+       //HI-SUM
+
+       l1t::EtSum towCount = l1t::EtSum();
```

Comment:
ok, so the data are 0 padded (as opposed to reading off the end of a data structure